### PR TITLE
Moved initialization of NodeInfo to SelenoidHelper

### PR DESCRIPTION
**Pull request by mreiche**
# Description

* Moves Selenium node detection to `testerra-selenoid-connector` module.
* Requires https://github.com/telekom/testerra-selenoid-connector/pull/6

### CODE REVIEW

File: driver-ui-desktop/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/DesktopWebDriverUtils.java [hunk 1]
```diff
@@ -21,42 +21,16 @@
...
-         */
-        try {
-            String nodeResponse = RESTUtils.requestGET(url + "/host/" + sessionId, 30 * 1000, String.class);
-            Gson gson = new GsonBuilder().create();
-            Map map = gson.fromJson(nodeResponse, Map.class);
-            double port = Double.parseDouble(map.get("Port").toString());
-            return new NodeInfo(map.get("Name").toString(), (int) port);
-        } catch (Exception e) {
-            log().error("Could not get node info: " + e.getMessage());
-            return new NodeInfo(seleniumUrl.getHost(), seleniumUrl.getPort());
```
Comment (martingrossmann) [inline driver-ui-desktop/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/DesktopWebDriverUtils.java:56]: As default (Testerra standalone) the Selenium URL and port should returned by this method.

File: driver-ui-desktop/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/DesktopWebDriverFactory.java [hunk 2]
```diff
@@ -324,14 +323,6 @@ private WebDriver newWebDriver(
             newDriver = startNewWebDriverSession(desktopWebDriverRequest, capabilities, remoteAddress, sessionContext);
         }
 
-        /*
-        Log User Agent and executing host
-         */
-        if (remoteAddress != null && newDriver instanceof RemoteWebDriver) {
```
Comment (martingrossmann) [inline driver-ui-desktop/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/DesktopWebDriverFactory.java:330]: See comment in `DesktopWebDriverUtils`.

**Comment by martingrossmann** [comment]
Is there a chance that ReportNG gets an update to show the node information?

**Comment by mreiche** [comment]
> Is there a chance that ReportNG gets an update to show the node information? Should a part of method details - session information.

**Comment by mreiche** [comment]
> Is there a chance that ReportNG gets an update to show the node information? Should a part of method details - session information.

Please create a feature ticket.

File: driver-ui-desktop/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/DesktopWebDriverUtils.java (hunk 1 above)
Comment (mreiche) [reply driver-ui-desktop/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/DesktopWebDriverUtils.java:56]: Please comment at telekom/testerra-selenoid-connector#6

File: driver-ui-desktop/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/DesktopWebDriverFactory.java (hunk 2 above)
Comment (mreiche) [reply driver-ui-desktop/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/DesktopWebDriverFactory.java:330]: Please comment at telekom/testerra-selenoid-connector#6

File: driver-ui-desktop/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/DesktopWebDriverUtils.java (hunk 1 above)
Comment (martingrossmann) [reply driver-ui-desktop/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/DesktopWebDriverUtils.java:56]: Yes, for the report update.
But I suggest to keep the selenium url stuff in Testerra as default information about the node.

Comment (mreiche) [reply driver-ui-desktop/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/DesktopWebDriverUtils.java:56]: The SessionContext contains actually both information: Selenium-Url and Node-Url.

**Review by erickubenka — APPROVED** [review]

**Review by martingrossmann — APPROVED** [review]

**Pull request merged**